products/statement-generator/src/components-layout/AppHeader.tsx

```diff
@@ -1,4 +1,4 @@
-import React, { useContext } from 'react';
+import React, { useContext, useState } from 'react';
```

Comment:
`useState` is not being used, right? Can you remove it?

Can you also run lint on this file.